OlinkAnalyze/R/Olink_anova.R

```diff
@@ -430,15 +447,13 @@ olink_anova_posthoc <- function(df,
       message(paste("Means estimated for each assay from ANOVA model: ",formula_string))
     }
 
-
-
     anova_posthoc_results <- df %>%
       dplyr::filter(OlinkID %in% olinkid_list) %>%
       dplyr::mutate(OlinkID = factor(OlinkID, levels = olinkid_list)) %>%
       dplyr::group_by(Assay, OlinkID, UniProt, Panel) %>%
-      dplyr::do(data.frame(summary(emmeans::emmeans(lm(as.formula(formula_string),data=.),
```

Comment:
why was summary removed here

Reply:
This is what caused the tukey-sidak warning